app/routes/geographic.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import APIRouter, HTTPException
+from typing import List, Optional
+from datetime import datetime
+import tempfile
+from pathlib import Path
+
+from app.services.geographic_analysis import GeographicAnalyzer, GeoLocation
+
+router = APIRouter()
+analyzer = GeographicAnalyzer()
+
+@router.get("/clusters")
```

Comment:
Can you define a ResponseModel for all of the new endpoints? Otherwise the Openapi docs won't know how to describe this endpoint.

https://fastapi.tiangolo.com/tutorial/response-model/